File: PSW-projekat/hospital-be-main/src/HospitalLibrary/Core/Model/Examination.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace HospitalLibrary.Core.Model
{
    public class Examination : BaseModel
    {
        public virtual Appointment Appointment { get; set; }
        public virtual ICollection<Prescription> Prescriptions { get; set; }
        public virtual ICollection<Symptom> Symptoms { get; set; }
        public String Report { get; set; }

        public Examination() { }

        public Examination(Appointment appointment, ICollection<Prescription> prescriptions, ICollection<Symptom> symptoms, string report)
        {
            Appointment = appointment;
            Prescriptions = prescriptions;
            Symptoms = symptoms;
            Report = report;
        }
    }
}

File: PSW-projekat/PZL-Angular/src/app/modules/hospital/model/examination.ts
import { Appointment } from "./appointment.model";
import { Prescription } from "./prescription";
import { Symptom } from "./symptom";

export class Examination {
    id: number;
    deleted: boolean;
    appointment: Appointment;
    prescriptions: Prescription[];
    symptoms: Symptom[];
    report: string;

    public constructor(id: any, deleted: any, appointment: any,
        prescriptions: any, symptoms: any, report:any) {
        this.id = id;
        this.deleted = deleted;
        this.appointment = appointment;
        this.prescriptions = prescriptions;
        this.symptoms = symptoms;
        this.report = report;

    }
}